bitcoin/src/psbt/map/output.rs

```diff
@@ -54,6 +55,11 @@ pub struct Output {
     /// Unknown key-value pairs for this output.
     #[cfg_attr(feature = "serde", serde(with = "crate::serde_utils::btreemap_as_seq_byte_values"))]
     pub unknown: BTreeMap<raw::Key, Vec<u8>>,
+
+    // Psbtv2 fields
+    pub amount: Option<Amount>,
```

Comment:
Incorrect doc-comment. Should start with `///` and mention the field. 
Should look something like this
```rust
// psbtv2 fields
/// <Amount description>
pub amount: Option<Amount>, 
```